src/server/pyproject.toml

```
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+packages = ['dcp_server']
+
+[tool.setuptools.dynamic]
+dependencies = {file = ["requirements.txt"]}
+
+[project]
+name = "data-centric-tool-server"
+version = "0.1"
```

Is this the right version? I think it is 0.2
```suggestion
version = "0.2"
```

---

this could be done dynamically instead.

---

created issue #11